test/tasks/after-suite.yaml

```diff
@@ -0,0 +1,26 @@
+# ginkgo-suite is a generic task that executes ginkgo on the given repository and ginkgo suite.
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: after-suite
+  namespace: karpenter-tests
+spec:
+  description: |
+    Executes after a test suite to clean up resources
+  params:
+    - name: cluster-name
+      description: Name of the cluster under test.
+  workspaces:
+    - name: ws
+  steps:
+
+  - name: delete-cluster
+    image: 767520670908.dkr.ecr.us-west-2.amazonaws.com/karpenter-test-infrastructure:v4
+    script: |
+      eksctl delete cluster --name $(params.cluster-name) --wait || true
+
+  - name: delete-iam-policies
```

Comment:
Should we reverse the order of these steps? IIUC we should delete in the reverse order that we create to ensure that there are no blocking dependencies.

Reply:
The IAM policies are in use by the cluster, which I believe will block deletion.

Reply:
You bring up a good point, if we have some instances that didn't get cleaned up in a test still using the policies, I believe it blocks.